Cisco/cisco-ise/tests/test_miscellaneous1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco ISE",
+        "dialect_uuid": "8a9894f8-d7bc-4c06-b96a-8808b3c6cade"
+      }
+    },
+    "message": "2023-06-07 04:26:17.306 +0200 60198 INFO null: MnT purge event occurred, MESSAGE=completed successfully,"
```

Comment:
Could you parse the "MESSAGE=completed successfully" and set [event.outcome](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html) with it?

Reply:
event.outcome doesn't seem to appear in the test, we'll see that later with @squioc